core/application/commands.go

```diff
@@ -37,6 +37,29 @@ func (a *app) CreateMicroVM(ctx context.Context, mvm *models.MicroVM) (*models.M
 		return nil, fmt.Errorf("an error occurred when attempting to validate microvm spec: %w", validErr)
 	}
 
+	if mvm.Spec.RootVolume.Source.VirtioFS != nil {
+		return nil, fmt.Errorf("VirtioFS is not available yet for RootVolume")
+	}
+	if len(mvm.Spec.AdditionalVolumes) > 0 {
+		virtioFSCount := 0 // Counter for VirtioFS volumes
+		
+		for _, volume := range mvm.Spec.AdditionalVolumes {
+			if volume.Source.Container != nil && volume.Source.VirtioFS != nil {
```

Comment:
Like above we could add a custom validator and attach it to the source field: https://github.com/liquidmetal-dev/flintlock/blob/main/core/models/volumes.go#L10

Reply:
Can you check if the most recent code update resolves this?